apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -183,3 +189,38 @@ var ShareDialog = React.createClass({
   }
 });
 module.exports = ShareDialog;
+
+if (BUILD_STYLEGUIDE) {
+  const fakei18n = {
+    t(s) {
+      return {
+      }[s] || s;
+    }
+  };
+
+  ShareDialog.styleGuideExamples = storybook => {
+    storybook
+      .storiesOf('ShareDialog', module)
+      .addStoryTable([
+        {
+          name: 'basic example',
+          story: () => <ShareDialog
+                           hideBackdrop={true}
+                           i18n={fakei18n}
+                           title="Share your project"
+                           shareCopyLink="Copy the link:"
+                           shareUrl="https://studio.code.org/projects/applab/GmBgH7e811sZP7-5bALAxQ"
+                           encodedShareUrl="some encoded url"
+                           closeText="Close"
+                           isAbusive={false}
+                           abuseTos="foo"
+                           abuseContact="bar"
+                           channelId="some-id"
+                           appType="applab"
+                           onClickPopup={storybook.action('onClickPopup')}
+                           onClickExport={storybook.action('onClickExport')}
+                       />
```

Comment:
attributes should be indented four spaces from the line above rather than from the start of the component. Alternatively, you can wrap the entire component in parens and put it all on a new line, i.e.
```
story: () => (
  <ShareDialog
      hideBackdrop={true}
      ...
  />
)